src/main/java/com/google/codeu/servlets/DeleteMessageServlet.java

```diff
@@ -27,6 +27,8 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import static java.lang.System.out;
```

Comment:
This import looks not necessary?

Reply:
Ah I used it for debugging but forgot to delete. It should be fixed by now.